src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,89 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+
+
+/**
+ * The Storage class handles the read and write of data from and to the local disk.
+ */
+public class Storage {
+    /**
+     * Data file path.
+     */
+    private final String filePath;
+
+    /**
+     * Instantiates a Storage object with a file path.
+     * @param path File path to read from and write to.
```

Comment:
maybe you can add a empty line between the description and the parameters? (as per the java coding standard - also applicable to the other javadoc comments!) :)